.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,48 @@
-^ Make sure to prefix the Title with [WIP]/[WIPTEST]
+<!-- Make sure to prefix the PR Title with any one of the below options...
 
-Purpose or Intent
-=================
+   [WIP] - Work in Progress
+   [WIPTEST] - Work in Progress with PRT Testing
+   [RFR] - Ready for 1st Review
+   [1LP][WIP] - 1st Level Pass, Work in Progress
+   [1LP][WIPTEST] - 1st Level Pass, Work in Progress with PRT Testing
+   [1LP][RFR] - 1st Level Pass, Ready for 2nd Review
+   [NOTEST] - No PRT testing (It will run smoke tests)
+-->
 
-__Fixing__ X because it is currently doing Y which is incorrect. It should be doing Z.
+## Purpose or Intent
+<!-- Please provide some information related to PR. Some examples are:
 
-or
+- __Fixing__ X because it is currently doing Y which is incorrect. It should be doing Z.
+- __Extending__ X because I need it to do Y so that I can update/add more test cases; PR #123 
+depends on it.
+- __Adding tests__ for feature X to cover cases Y and Z. They were implemented in product 
+version1.2.3.
+- __Updating tests__ for feature X because the behavior changed in product version 1.2.3.
+- __Enhancement__ for feature X
 
-__Extending__ X because I need it to do Y so that I can update / add more test cases; PR #123 depends on it.
+Note: You can introduce Screenshots/Gifs for providing more information.
+-->
 
-or
+### PRT Run
+<!-- Try to run tests for each PR or prefix the PR Title with [NOTEST]. Some examples are:
```

Comment:
I would focus less on documenting pytest kwargs here, and more documenting what PRT actually does.  Please include some notes about the following things:

1. Its an internal RH service, and only runs against signed and whitelisted commits.  
2. Contributors should contact the maintainers if they need guidance or assistance with PRT results.
3. That default run is `-m smoke`
4. That it runs against a single appliance

There are upcoming changes to PRT's architecture and what it will support - we'll update this as those changes are made.

Thanks so much for making this information more visible by putting it in the template, great idea @digitronik

Reply:
I do think it is helpful to have those examples below.

Reply:
i like the examples too

Reply:
@mshriver @kedark3 @jawatts so I added both... I like Mike suggestion to give some basic info about PRT also I like to add some examples because new contributors confused and still i found some contributor unable to collect specific tests form multiple files so lets add 5 extra lines.